### DISCUSSION TITLE
World generator

### DISCUSSION THREAD

**Discussion by Bananaguns2** [misode/misode.github.io#670] [Q&A]
Is it my internet connection or the tool?[https://misode.github.io/dimension/?version=1.21.4&preset=overworld](https://misode.github.io/dimension/?version=1.21.4&preset=overworld) takes a super long time to load & is super laggy just scrolling through the code 🤦‍♂️ it gets so bad that my web browser makes me click ok wait for website to load button like 4 times before any website changes take place I assume its because there's almost 2,000 biome entries? Why is there so many it looks like alot of the entries are duplicates are they really necessary for minecraft to properly function in vanilla without modification?

**Comment by misode** [comment]
Yeah this is a common issue. See #651
